src/browser-frontend/actions/pages-model-actions.js

```diff
@@ -27,6 +27,7 @@ export default createActions({
   TABBAR: {
     SET_TAB_STATE: identity,
     CLOSE_TAB_ANIMATED: identity,
+    CHANGE_OPTIONAL_CLASS: identity,
```

Comment:
Without knowing the code, this action's name would make it seem that you're changing some optional class on the tabbar itself. Maybe renaming this to `CHANGE_OPTIONAL_TAB_CLASS` would make it more obvious.